src/node.cc

```diff
@@ -47,6 +47,7 @@
 #include "node_version.h"
 
 #if HAVE_OPENSSL
+#include "ncrypto.h"
```

Comment:
I think we should add this check to every `ncrypto.h` include, in this PR (and possibly in the future too.

Reply:
We can do so as we go.